test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class SortCommandTest {
+    private TypicalPersons td = new TypicalPersons();
+
+    private AddressBook typicalAddressBook = td.getTypicalAddressBook();
+    private List<ReadOnlyPerson> listWithAllTypicalPersons = Arrays.asList(td.getTypicalPersons());
+    private List<ReadOnlyPerson> listWithInorderedTypicalPersons = Arrays.asList(td.amy, td.candy, td.bill, td.dan);
+
+    @Test
+    public void execute_invalidIndex_returnsInvalidIndexMessage() {
```

Comment:
The naming of this method seems a little unclear - are you testing for an invalid index?